src/components/Executions/constants.ts

```diff
@@ -96,6 +96,11 @@ export const nodeExecutionPhaseConstants: {
         text: 'Running',
         textColor: secondaryTextColor
     },
+    [NodeExecutionPhase.DYNAMIC_RUNNING]: {
+        badgeColor: statusColors.RUNNING,
+        text: 'Running',
```

Comment:
Should it read something different? Maybe

Reply:
I wasn't sure about this, not sure what the best text would be to convey the status to a user but still concise. I can ask Sandra or Hongxun perhaps